core-build/gulp-core-build/src/logging.ts

```diff
@@ -183,6 +182,7 @@ function writeSummary(callback: () => void): void {
 
     // flush the log
     afterStreamsFlushed(() => {
+      const shouldRelogIssues: boolean = getFlagValue('relogIssues');
```

Comment:
>const shouldRelogIssues: boolean = getFlagValue('relogIssues'); [](start = 6, length = 63)

doesn't it already re-log all the issues without this flag?

Reply:
The default value for that flag is "true"